src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -33,7 +33,12 @@
      */
     public AddCommand parse(String args) throws ParseException {
         requireNonNull(args);
-        PersonType personType = ParserUtil.parsePersonType(args);
+        PersonType personType;
+        try {
+            personType = ParserUtil.parsePersonType(args);
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE), pe);
+        }
```

Comment:
I think this part might be unnecessary? Cause the only issue here is just the person type wrong right? In `PersonType.java` it already throws a ParseException with the message of "The given person type is not t or s." Is that sufficient?